static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -37,7 +37,7 @@ export function OverviewTimeline({monitorList}: Props) {
   );
 
   return (
-    <MonitorListPanel>
+    <MonitorListPanel rowCount={monitorList.length}>
```

Comment:
if we call it rowCount should we just add the +1 in here? `rowCount={monitorList.length + 1}` or just change the name of the prop to `monitorCount`?

Reply:
I like it